### PR TITLE
Add Walavouchey to the list of wiki maintainers

### PR DESCRIPTION
**Pull request by TicClick**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

File: wiki/osu!_wiki/Maintenance/List_of_maintainers/en.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ If you need help with making a contribution or want a review on your pull reques
 | ![][flag_FR] [Shiro](https://osu.ppy.sh/users/113005) | [osuShiro](https://github.com/osuShiro) | Inactive | French |
 | ![][flag_CH] [TicClick](https://osu.ppy.sh/users/672931) | [TicClick](https://github.com/TicClick) | **Active** | Russian |
 | ![][flag_PL] [TPGPL](https://osu.ppy.sh/users/3944705) | [TPGPL](https://github.com/TPGPL) | **Active** | Polish |
+| ![][flag_SE] [Walavouchey](https://osu.ppy.sh/users/5773079) | [Walavouchey](https://github.com/Walavouchey) | **Active** | Swedish |
```
Comment (Walavouchey) [inline wiki/osu!_wiki/Maintenance/List_of_maintainers/en.md:25]:
```suggestion
| ![][flag_SE] [Walavouchey](https://osu.ppy.sh/users/5773079) | [Walavouchey](https://github.com/Walavouchey) | **Active** | Finnish, Swedish |
```

File: wiki/osu!_wiki/Maintenance/List_of_maintainers/id.md [hunk 2]
```diff
@@ -20,6 +20,7 @@ Jika kamu memerlukan bantuan untuk memberikan kontribusi atau ingin *pull reques
 | ![][flag_FR] [Shiro](https://osu.ppy.sh/users/113005) | [osuShiro](https://github.com/osuShiro) | Tidak aktif | Prancis |
 | ![][flag_CH] [TicClick](https://osu.ppy.sh/users/672931) | [TicClick](https://github.com/TicClick) | **Aktif** | Rusia |
 | ![][flag_PL] [TPGPL](https://osu.ppy.sh/users/3944705) | [TPGPL](https://github.com/TPGPL) | **Aktif** | Polandia |
+| ![][flag_SE] [Walavouchey](https://osu.ppy.sh/users/5773079) | [Walavouchey](https://github.com/Walavouchey) | **Actif** | Swedia |
```
Comment (ZeroPyrozen) [inline wiki/osu!_wiki/Maintenance/List_of_maintainers/id.md:23]:
```suggestion
| ![][flag_SE] [Walavouchey](https://osu.ppy.sh/users/5773079) | [Walavouchey](https://github.com/Walavouchey) | **Aktif** | Swedia |
```

File: wiki/osu!_wiki/Maintenance/List_of_maintainers/en.md (hunk 1 above)
Comment (TicClick) [reply wiki/osu!_wiki/Maintenance/List_of_maintainers/en.md]: added

File: wiki/osu!_wiki/Maintenance/List_of_maintainers/id.md (hunk 2 above)
Comment (TicClick) [reply wiki/osu!_wiki/Maintenance/List_of_maintainers/id.md]: fied, thanks. was a carryover from French

**Review by peppy — APPROVED** [review]

**Pull request merged**